### PR TITLE
SecurityStrategy supporting 403 (#219)

### PR DESCRIPTION
**Pull request by fschlesinger**
Proposed changes to add support for logging requests and responses with HTTP 403 status in the default SpringBoot setup, as discussed in  #219.

### CODE REVIEW

File: logbook-servlet/src/main/java/org/zalando/logbook/servlet/SecurityStrategy.java [hunk 1]
```diff
@@ -47,7 +47,8 @@ public void doFilter(final Logbook logbook, final HttpServletRequest httpRequest
     }
 
     private boolean isUnauthorized(final HttpServletResponse response) {
```
Comment (whiskeysierra) [inline logbook-servlet/src/main/java/org/zalando/logbook/servlet/SecurityStrategy.java]: `isUnauthorizedOrForbidden`?

**Comment by whiskeysierra** [comment]
> Is there any documentation to update?

I don't think so.

File: logbook-servlet/src/main/java/org/zalando/logbook/servlet/SecurityStrategy.java (hunk 1 above)
Comment (fschlesinger) [reply logbook-servlet/src/main/java/org/zalando/logbook/servlet/SecurityStrategy.java]: fixed

**Comment by whiskeysierra** [comment]
:+1:

**Comment by whiskeysierra** [comment]
@fschlesinger Thanks for the contribution :tada: I released the fix as 1.7.2

**Pull request merged**